contextualized/networks/__init__.py

```diff
@@ -0,0 +1,41 @@
+from contextualized.modules import MLP, NGAM
+ENCODERS = {
+    'mlp': MLP,
+    'ngam': NGAM,
+}
+
+from contextualized.functions import *
+LINK_FUNCTIONS = {
+    'identity': linear_link_constructor(),
+    'logistic': logistic_constructor(),
+    'softmax': softmax_link_constructor()
+}
+
+from contextualized.regression.datasets import UnivariateDataset, MultitaskUnivariateDataset
```

Comment:
If these are used for both regression and networks, should they be in contextualized.regression?